process-instance-migration/src/main/frontend/src/styles/settings.scss

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
here is no content

Reply:
This is a generated file, we might need it later